op-e2e/e2eutils/setup.go

```diff
@@ -235,6 +239,7 @@ func SystemConfigFromDeployConfig(deployConfig *genesis.DeployConfig) eth.System
 }
 
 func ApplyDeployConfigForks(deployConfig *genesis.DeployConfig) {
+	isIsthmus := os.Getenv("OP_E2E_USE_ISTHMUS") == "true"
 	isHolocene := os.Getenv("OP_E2E_USE_HOLOCENE") == "true"
```

Comment:
following the convention of using a future fork to override:
```suggestion
	isHolocene := isIsthmus || os.Getenv("OP_E2E_USE_HOLOCENE") == "true"
```